core/contractsdk/cpp/example/short_content.cc

```diff
@@ -0,0 +1,100 @@
+#include "xchain/xchain.h"
+
+const std::string UserBucket = "USER";
+
+// 短内容存证API规范
+// 参数由Context提供
+class ShortContentDepositBasic {
+public:
+    // 初始化，基本不做任何工作
+    virtual void initialize() = 0;
+    // 将用户参数{user_id,title,topic}存储到磁盘
+    virtual void storeShortContent() = 0;
+    // 按照用户粒度查询内容
+    virtual void queryByUser() = 0;
+    // 按照标题粒度查询内容，用户名是必须的
+    virtual void queryByTitle() = 0;
+    // 按照标题粒度查询内容，用户名是必须的
+    virtual void queryByTopic() = 0;
+};
+
+struct ShortContentDeposit : public ShortContentDepositBasic, public xchain::Contract {
+public:
+    void initialize() {
+        xchain::Context* ctx = this->context();
+        ctx->ok("initialize success");
+    }
+    void storeShortContent() {
+        xchain::Context* ctx = this->context();
+        std::string user_id = ctx->arg("user_id");
+        std::string title = ctx->arg("title");
+        std::string topic = ctx->arg("topic");
+        std::string content = ctx->arg("content");
+        const std::string userKey = UserBucket + "/" + user_id + "/" + topic + "/" + title;
```

Comment:
suggest limit the length of topic and title, this is a short content storage contract